flask_authz/casbin_enforcer.py

```diff
@@ -85,6 +89,9 @@ def wrapper(*args, **kwargs):
                                 "decoding is unsupported by flask-casbin at this time"
                             )
                             continue
+
+                        if self.user_name_headers and header in self.user_name_headers:
```

Comment:
Per http protocol headers are not case sensitive, perhaps we should ensure we support that here, or when we load by forcing everything .lower()?

Reply:
Good call out. Let me look into what flask is doing with the headers in the request. They might be doing a .upper() on it.

Reply:
added changes to remove case sensitivity